bats/fb-content-katello.bats

```diff
@@ -121,6 +121,10 @@ EOF
   tPackageExists subscription-manager || tPackageInstall subscription-manager
 }
 
+@test "disable puppet agent to prevent checkin from registering host to another org" {
+  run systemctl stop puppet
```

Comment:
This will fail if Puppet is not present and break. Probably needs some `systemctl is-active` love